LeaderboardBackend/Controllers/UsersController.cs

```diff
@@ -47,4 +49,45 @@ public async Task<ActionResult<UserViewModel>> Me() => (await userService.GetUse
             badCredentials => Unauthorized(),
             userNotFound => NotFound()
         );
+
+    [Authorize(Policy = UserTypes.ADMINISTRATOR)]
+    [HttpPatch("users/{id}")]
+    [SwaggerOperation(
+        "Updates a user. This request is restricted to administrators, and currently " +
+        "only for banning/unbanning users.",
+        OperationId = "updateUser"
+    )]
+    [SwaggerResponse(204)]
+    [SwaggerResponse(401)]
+    [SwaggerResponse(
+        403,
+        "This request is not an attempt by an admin to ban/unban confirmed users.",
```

Comment:
This is confusing. I would write something like "The request was not sent by an admin, the target user is an admin, or the role provided was not BANNED or CONFIRMED." That makes it crystal clear what the scenarios that can trigger 403 are.